api/app/GraphQL/Mutations/DownloadNominationsCsv.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\GraphQL\Mutations;
+
+use App\Generators\NominationsCsvGenerator;
+use App\Jobs\GenerateUserFile;
+use App\Models\TalentNominationEvent;
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Validation\UnauthorizedException;
+
+final class DownloadNominationsCsv
+{
+    /**
+     * Dispatches the generation of a
+     * csv containing community interests
+     *
+     * @disregard P1003 We are not going to be using this var
+     */
+    public function __invoke($_, array $args)
+    {
+
+        $user = Auth::user();
+        throw_unless(is_string($user?->id), UnauthorizedException::class);
+
+        $ids = $args['ids'] ?? null;
+        $talentNominationEventId = $args['talentNominationEventId'] ?? null;
+        $talentNominationEvent = TalentNominationEvent::findOrFail($talentNominationEventId);
+        $talentNominationEventName = App::getLocale() === 'en' ? $talentNominationEvent->name['en'] : $talentNominationEvent->name['fr'];
+
+        try {
+            $generator = new NominationsCsvGenerator(
+                fileName: sprintf('%s%s_%s', str_replace(' ', '', $talentNominationEventName), __('.nominations'), date('Y-m-d_His')),
```

Comment:
Is this translation working?  It looks the same in English and French.  Maybe it's the same word but I don't see it in the lang file diffs.
<img width="582" height="265" alt="image" src="https://github.com/user-attachments/assets/e631baca-c778-4bfc-a6bd-3ce58b7ed1a2" />

Reply:
Fixed in [Fix file name again](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13988/commits/7a59612b1db9271df0d4b73c67cbff9fc7eb993b)